libtextsecure/sendmessage.js

```diff
@@ -550,57 +529,60 @@ MessageSender.prototype = {
     });
   },
 
-  sendRequestConfigurationSyncMessage(options) {
+  sendRequestConfigurationSyncMessage() {
     const myNumber = textsecure.storage.user.getNumber();
     const myDevice = textsecure.storage.user.getDeviceId();
     if (myDevice !== 1 && myDevice !== '1') {
-      const request = new textsecure.protobuf.SyncMessage.Request();
-      request.type = textsecure.protobuf.SyncMessage.Request.Type.CONFIGURATION;
-      const syncMessage = this.createSyncMessage();
-      syncMessage.request = request;
-      const contentMessage = new textsecure.protobuf.Content();
-      contentMessage.syncMessage = syncMessage;
+      const { CONFIGURATION } = textsecure.protobuf.SyncMessage.Request.Type;
+      const user = libsession.Types.PubKey.from(myNumber);
+      const { RequestContactSyncMessage } = window.libsession.Messages.Outgoing;
 
-      const silent = true;
-      return this.sendIndividualProto(
-        myNumber,
-        contentMessage,
-        Date.now(),
-        silent,
-        options
-      );
+      const requestContactSyncMessage = new RequestContactSyncMessage({
+        timestamp: Date.now(),
+        reqestType: CONFIGURATION,
+      });
+      return libsession.getMessageQueue().send(user, requestContactSyncMessage);
```

Comment:
```suggestion
      return libsession.getMessageQueue().sendSyncMessage( requestContactSyncMessage);
```
I know that in the future we should probably have a way to send a sync message to a specific device but for now this should be fine.